libsrc/meshing/meshclass.cpp

```diff
@@ -7302,6 +7318,13 @@ namespace netgen
       surfelementht->PrintMemInfo (cout);
   }
 
+  void Mesh :: SetMeshName(std::string name){
+    meshname = name;
+  }
+  std::string Mesh :: GetMeshName(){
+    return meshname;
+  }
+
```

Comment:
What is the meshname for?

Reply:
Is a parameter used in Firedrake, so I added it also here.